src/JSInterop/Microsoft.JSInterop/src/Infrastructure/DotNetDispatcher.cs

```diff
@@ -348,6 +361,16 @@ private static (MethodInfo, Type[]) GetCachedMethodInfo(AssemblyKey assemblyKey,
         }
     }
 
+    private static Task GetTaskByType(Type type, object obj)
+    {
+        var assemblyMethods = _cachedConvertToTaskByType.GetOrAdd(type, (Type t)=>
+            _taskConverterMethodInfo.MakeGenericMethod(t).CreateDelegate<Func<object, Task>>());
+
+        return assemblyMethods.Invoke(obj);
+    }
+    private static readonly MethodInfo _taskConverterMethodInfo = typeof(DotNetDispatcher).GetMethod(nameof(CreateValueTaskConverter), BindingFlags.NonPublic | BindingFlags.Static)!;
+    private static Task CreateValueTaskConverter<T>(object? result) => ((ValueTask<T>)result!).AsTask();
```

Comment:
No need for the `?` here nor the `!` in the method body, since you already know the object is not null

Reply:
I think the T there needs `[DynamicallyAccessedMembers(LinkerFlags.JsonSerialized)] T`

Reply:
Unfortunately CI still shows errors. By the way is there a way to do CI-like code analysis locally?

Reply:
@senbar I believe if you do eng\build.cmd or any other build.cmd in the appropriate subproject it will produce the error.

Reply:
> do you know who can help us figure out the annotation we need to use here for linking


I believe it may just be a matter of doing the following?

https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebAssembly/testassets/WasmLinkerTest/README.md#trimmer-baseline-verification

Reply:
> > do you know who can help us figure out the annotation we need to use here for linking
> 
> I believe it may just be a matter of doing the following?
> 
> https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebAssembly/testassets/WasmLinkerTest/README.md#trimmer-baseline-verification

Yes! The errors are gone, thanks 👍 

> @senbar I believe if you do eng\build.cmd or any other build.cmd in the appropriate subproject it will produce the error.

unfortunately it doens't produce these errors on build